prepare_llm_models.sh

```diff
@@ -68,3 +85,24 @@ if [ -d "$1/$RERANK_MODEL" ]; then
 else
   python3 demos/common/export_models/export_model.py rerank --source_model "$RERANK_MODEL" --weight-format int8 --model_repository_path $1
 fi
+
+if [ -d "$1/$QWEN3_MODEL" ]; then
+  echo "Models directory $1/$QWEN3_MODEL exists. Skipping downloading models."
+else
+  mkdir -p $1/$QWEN3_MODEL
+  convert_tokenizer $QWEN3_MODEL --with_detokenizer -o $1/$QWEN3_MODEL
```

Comment:
where to take convert_tokenizer from?

Reply:
It comes with openvino-tokenizers which is a dependency of optimum. 
If you have installed packages required by `export_model.py`, you already have `convert_tokenizer`